java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -278,6 +316,29 @@ private void updateConfig(Properties prop, String key, HashSet<String> hs) {
     }
   }
 
+  private void addMethodPropertiesToConfigMethodProperties(
+      Set<Map<String, Object>> methodProperties) {
+    Iterator<Map<String, Object>> methodPropsIterator = methodProperties.iterator();
+    while (methodPropsIterator.hasNext()) {
+      Map<String, Object> methodProperty = methodPropsIterator.next();
+      if (checkValueStringExistsAndNotEmpty(methodProperty, FLAG_TYPE_KEY)
+          && checkValueStringExistsAndNotEmpty(methodProperty, METHOD_NAME_KEY)) {
+        addMethodPropertyToConfigMethodProperties(methodProperty);
```

Comment:
given the update to `checkValueStringExistsAndNotEmpty`, this code can be simplified further. Also, 
`addMethodPropertyToConfigMethodProperties` is only called here and can be confusing with the current method `addMethodPropertiesToConfigMethodProperties`. Can we inline `addMethodPropertyToConfigMethodProperties`?